ONSdigital/eq-survey-runner
# eq-520 Fieldset/legend

### What is the context of this PR?

[Trello card](https://trello.com/c/TTUzhhCq/520-2-improve-markup-for-fieldset-and-legend-etc)

This PR adds `fieldset` and `legend` markup to radio button/checkbox questions to improve document semantics and therefore accessibility of the questions/answers.

It also fixes some issues where the validity of HTML was affecting the fieldset\legend being actually parsed correctly by clients.
### How to review
1. Visit Star Wars survey
2. Navigate to first section "Choose a side"
3. Using Voiceover, confirm that the user is informed that they are inside a "group" (`fieldset`) when navigating to the radio button controls. 
4. Confirm that you are able to navigate outside the group (ctrl+option+shift+up) and that it reads the `legend` correctly ("Which side of the force are you on?")
